crates/pop-cli/src/commands/install/mod.rs

```diff
@@ -0,0 +1,313 @@
+// SPDX-License-Identifier: GPL-3.0
+use crate::style::{style, Theme};
+use anyhow::Context;
+use clap::Args;
+use cliclack::{clear_screen, confirm, intro, log, outro, set_theme};
+use duct::cmd;
+use os_info::Type;
+use strum::Display;
+use tokio::{fs, process::Command};
+use Dependencies::*;
+
+#[derive(Display)]
+pub enum Dependencies {
+	#[strum(serialize = "build-essential")]
+	BuildEssential,
+	#[strum(serialize = "clang")]
+	Clang,
+	#[strum(serialize = "clang-devel")]
+	ClangDevel,
+	#[strum(serialize = "cmake")]
+	Cmake,
+	#[strum(serialize = "curl")]
+	Curl,
+	#[strum(serialize = "gcc")]
+	Gcc,
+	#[strum(serialize = "git")]
+	Git,
+	#[strum(serialize = "homebrew")]
+	Homebrew,
+	#[strum(serialize = "libclang-dev")]
+	LibClang,
+	#[strum(serialize = "libssl-dev")]
+	Libssl,
+	#[strum(serialize = "make")]
+	Make,
+	#[strum(serialize = "openssl")]
+	Openssl,
+	#[strum(serialize = "openssl-devel")]
+	OpenSslDevel,
+	#[strum(serialize = "pkg-config")]
+	PkgConfig,
+	#[strum(serialize = "protobuf")]
+	Protobuf,
+	#[strum(serialize = "protobuf-compiler")]
+	ProtobufCompiler,
+	#[strum(serialize = "rustup")]
+	Rustup,
+}
+const MACOS_PACKAGES: [Dependencies; 5] = [Homebrew, Protobuf, Openssl, Rustup, Cmake];
```

Comment:
I thought we werent using this?

Reply:
I apologize, this was not meant to be pushed.